cobbler/cobblerd.py

```diff
@@ -30,7 +30,7 @@
 from cobbler.api import CobblerAPI
 
 # FIXME: Fix the absolute path and logging dependency on it.
```

Comment:
You may remove the part on the logging dependency since you removed this.

Reply:
Amended to the last commit. Thanks!